packages/element/src/itk-view-2d.ts

```diff
@@ -0,0 +1,84 @@
+import { LitElement, css, html } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { ref } from 'lit/directives/ref.js';
+import { ContextConsumer } from '@lit/context';
+
+import {
+  createView2d,
+  View2dVtkjsActor,
+  createView2dVtkjs,
+} from '@itk-viewer/vtkjs/view-2d-vtkjs.js';
+import { type Viewer, viewerContext } from './viewer-context.js';
+
+@customElement('itk-view-2d')
```

Comment:
rename to `itk-view-2d-vtkjs` and module name

Reply:
Renamed.